frontend/game-rating-platform/src/compoments/GameTable.tsx

```diff
@@ -81,48 +95,66 @@ export function GameTable({
   });
 
   return (
-    <table className="table mx-auto">
-      <thead>
-        <tr>
-          <th>Titel</th>
-          <th>Erscheinungsdatum</th>
-          <th>Entwickler</th>
-          <th>Bewertung</th>
-        </tr>
-      </thead>
-      <tbody>
-        {sortedProps.map<JSX.Element>((prop) => (
-          <GameRow
-            id={prop.id}
-            title={prop.title}
-            releaseDate={prop.releaseDate}
-            developers={prop.developers}
-            avgRating={prop.avgRating}
-            platform={prop.platform}
-          />
-        ))}
-      </tbody>
-    </table>
+    <>
+      <table className="table mx-auto">
+        <thead>
+          <tr>
+            <th>Titel</th>
+            <th>Erscheinungsdatum</th>
+            <th>Entwickler</th>
+            <th>Bewertung</th>
+          </tr>
+        </thead>
+        <tbody>
+          {sortedProps.map<JSX.Element>((prop) => (
+            <GameRow
+              title={prop.title}
+              releaseDate={prop.releaseDate}
+              developers={prop.developers}
+              avgRating={prop.avgRating}
+              onClickHandler={openModal}
+            />
+          ))}
+        </tbody>
+      </table>
+      <GameDetails
+        gameData={clickedGame}
+        active={activeModal}
+        closeHandler={closeModal}
+      />
+    </>
   );
 }
 
+interface GameRowProps {
+  avgRating: number;
+  developers: string[];
+  releaseDate: string;
+  title: string;
+  onClickHandler: (e: SyntheticEvent<HTMLTableRowElement, Event>) => void;
+}
+
 function GameRow({
-  id,
   avgRating,
   developers,
   releaseDate,
   title,
-}: GameProps): JSX.Element {
-  const ratingScore = Number.isNaN(Number(avgRating)) ? 0 : avgRating;
+  onClickHandler,
+}: GameRowProps): JSX.Element {
   const developerStr =
     developers !== undefined ? developers.join(", ") : undefined;
 
   return (
-    <tr>
+    <tr
+      onClick={onClickHandler}
+      id={title}
```

Comment:
Warum ist der `title` die `id` und nicht die `id`? 🤔

Und in Zeile 34 dann auch nicht `prop.title === e.currentTarget.id` sondern `prop.id === e.currentTarget.id`. Ist doch viel intuitiver, oder?